x-pack/docs/en/rest-api/security/create-roles.asciidoc

```diff
@@ -24,8 +24,13 @@ privilege.
 [[security-api-put-role-desc]]
 ==== {api-description-title}
 
-The role API is generally the preferred way to manage roles, rather than using
-file-based role management. For more information about the native realm, see 
+The role management APIs are generally the preferred way to manage roles, rather
+than using
+{stack-ov}/defining-roles.html#roles-management-file[file-based role management].
+This create or update roles API cannot update roles that are defined in roles
```

Comment:
> This ... API cannot update roles that are defined in roles files.

I made this change since it seemed like a logical extrapolation from the other updates.  If it's not correct, however, we can remove it.